src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -40,6 +38,7 @@ const propTypes = {
 
     /* Onyx Props */
     /** List of reports */
+    // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
This goes back to one of the things that @marcaaron and I were discussing in [this big thread above](https://github.com/Expensify/App/pull/10800#discussion_r970764202)... but if we remove them, then it prevents the options from re-rendering to show up-to-date data when values are updated in Onyx (since the options aren't using `withOnyx()` and instead relying on `Onyx.connect()` in `SidebarUtils.js`.

Reply:
Ah ok. Thanks for explaining that. As a gut reaction I don't like it, but I'll start reading that thread and see what I think.

Reply:
Now it makes sense why we are doing it this way and it sounds like we have a good plan for trying to clean this up in the future.